server/models/Favorite.js

```diff
@@ -0,0 +1,40 @@
+import pool from '../database.js'
+
+export default class Coin {
+    id;
+    user_id;
+    coins_ids;
+
+    constructor(row) {
+        this.id = row.id;
+        this.user_id = row.user_id;
+        this.coins_ids = row.coins_ids;
+    }
+
+    static async getAll() {
+        try {
+            const { rows } = await pool.query('SELECT * FROM users_favorite_coins');
```

Comment:
Implement a check here to not get an error. Something like if (rows[0]) return... else return some Error